flake8_trio/visitors/visitor91x.py

```diff
@@ -590,3 +721,77 @@ def visit_CompFor(self, node: cst.CompFor):
     # ignore their content
     def visit_GeneratorExp(self, node: cst.GeneratorExp):
         return False
+
+    def visit_Import(self, node: cst.Import):
+        """Register explicitly imported library.
+
+        This is *slightly* different enough from what the utility visitor does
+        that it's added here, but the functionality is maybe better placed in there.
```

Comment:
it'd probably just need another bit saved in self.library on whether it's been explicitly imported, or just assumed/specified with `--anyio`. But I'll procrastinate on that with the on-the-spot-made-up-reasoning that there's ast visitors that use the current self.library functionality and I neither wanna update both nor have different library logic between them.